test/Microsoft.Health.Fhir.Tests.Integration/Features/Operations/CosmosFhirOperationsDataStoreTests.cs

```diff
@@ -0,0 +1,199 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+using MediatR;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Health.Fhir.Core.Extensions;
+using Microsoft.Health.Fhir.Core.Features.Operations;
+using Microsoft.Health.Fhir.Core.Features.Operations.Export;
+using Microsoft.Health.Fhir.Core.Features.Operations.Export.Models;
+using Microsoft.Health.Fhir.Core.Messages.Export;
+using Microsoft.Health.Fhir.Tests.Common.FixtureParameters;
+using Microsoft.Health.Fhir.Tests.Integration.Persistence;
+using Xunit;
+
+namespace Microsoft.Health.Fhir.Tests.Integration.Features.Operations
+{
+    [FhirStorageTestsFixtureArgumentSets(DataStore.CosmosDb)]
+    public class CosmosFhirOperationsDataStoreTests : IClassFixture<FhirStorageTestsFixture>, IAsyncLifetime
```

Comment:
Seems like these are data provider-agnostic. Should probably drop CosmosDb from the test class name

Reply:
Good call. Updated the name.